source/scan.js

```diff
@@ -22,7 +24,7 @@ import _curry3 from './internal/_curry3.js';
  *      const factorials = R.scan(R.multiply, 1, numbers); //=> [1, 1, 2, 6, 24]
  * @symb R.scan(f, a, [b, c]) = [a, f(a, b), f(f(a, b), c)]
  */
-var scan = _curry3(function scan(fn, acc, list) {
+var scan = _curry3(_dispatchable([], _xscan, function scan(fn, acc, list) {
```

Comment:
If I understand this correctly then `R.scan` can act as a transducer when given a transformer as a list. Perhaps we should mention that in the description?

Reply:
Thanks for the observation, @customcommander. I've added that in the description.